app/controllers/api/avatars.php

```diff
@@ -123,7 +123,7 @@
     ->desc('Get Country Flag')
     ->groups(['api', 'avatars'])
     ->label('scope', 'avatars.read')
-    ->label('sdk.platform', [APP_PLATFORM_CLIENT, APP_PLATFORM_SERVER])
+    ->label('sdk.security', [APP_AUTH_TYPE_SESSION, APP_AUTH_TYPE_KEY, APP_AUTH_TYPE_JWT])
```

Comment:
As said on Discord before, security might be misleading here - since this endpoint is open to guests as well.

Reply:
Replaced sdk.security with sdk.auth